src/main.rs

```diff
@@ -602,6 +641,13 @@ fn main() {
       (about: "prints the parsed configuration file")
       (@arg DEFAULT: --default "print the default configuration")
     )
+    (@subcommand info =>
+      (about: "implementation of repo info")
+      (@arg DIFF: -d --diff "show full info and commit diff including remote branches")
+      //(@arg OVERVIEW: -o --overview "show overview of all local commits")
+      (@arg BRANCH: -b --current-branch "consider only checked out branches")
+      (@arg LOCAL: -l --local-only "disable all remote operations")
```

Comment:
This needs to be --("local-only"), or else it gets interpreted as --local (I need to get around to removing the use of the clap macros...)